lib/LWP/ConsoleLogger.pm

```diff
@@ -383,6 +381,12 @@ sub _parse_body {
         }
         catch { $t->row("Error parsing JSON: $_") };
     }
+    elsif ( $type && $type eq 'application' && $subtype eq 'javascript' ) {
+
+        # clean it up a bit, and print some of it
+        $content =~ s{^\s*}{}mg;
+        $content = substr( $content, 0, 252 ) . '...';
```

Comment:
Probably we should only append the ellipsis if we're sure the content is being truncated.